hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1085,21 +1085,30 @@ def hpat_pandas_series_prod(self, axis=None, skipna=None, level=None, numeric_on
 
     Parameters
     -----------
-    axis: {index (0)}
-        *unsupported* Axis for the function to be applied on.
-    skipna: :obj:`bool` object, default :obj:`True`
+    self: :obj:`pandas.Series`
+        input series
+    axis: :obj:`int`
```

Comment:
Actually axis could be a string (`'index'` or `'columns'`), not only an integer. You could add string type here. Moreover you could check on typing stage of the implementation that axis type is able to be a string.

Reply:
I have created https://github.com/IntelPython/hpat/issues/235 for discussing documentation for parameters.